app/views/products/_product_form.html.erb

```diff
@@ -12,15 +12,16 @@
           <fieldset id="certification_options">
             <legend class="control-label">Certification Types</legend>
             <div id="certifications_errors_container"></div>
-            <%= f.form_group help: "You must select C1, C2, or C4 in order to utilize Cypress for testing" do %>
+
+            <%= f.form_group help: "You must select C1, C2, C3, or C4 in order to utilize Cypress for testing" do %>
```

Comment:
I feel like we can remove this help message now that they're all viable by themselves